stream_alert/rule_processor/handler.py

```diff
@@ -56,30 +70,46 @@ def run(self, event):
         Returns:
             [boolean] True if all logs being parsed match a schema
         """
-        LOGGER.debug('Number of Records: %d', len(event.get('Records', [])))
-
-        config = load_config()
+        records = event.get('Records', [])
+        LOGGER.debug('Number of Records: %d', len(records))
+        if not records:
+            return False
+
+        for raw_record in records:
+            # Get the service and entity from the payload. If the service/entity
+            # is not in our config, log and error and go onto the next record
+            service, entity = self.classifier.extract_service_and_entity(raw_record)
+            if not service:
+                LOGGER.error('No valid service found in payload\'s raw record')
+
+            if not entity:
+                LOGGER.error(
+                    'Unable to map entity from payload\'s raw record for service %s',
+                    service)
+
+            if not (service and entity):
```

Comment:
Why not just `continue` in each `service`/`entity` check above?  You could also print the raw record there too and then just omit this block.